tardis/analysis_tools/opacities.py

```diff
@@ -1,10 +1,11 @@
 """This module provides an opacity calculator class with which the opacities
 and optical depth information may be extracted from Tardis runs."""
+
 import logging
 import numpy as np
 import astropy.units as units
-from tardis import constants as const
```

Comment:
Definitely need to use this constants module for consistency with the rest of TARDIS

Reply:
Reverting back to tardis const package